docs/django-admin/roles.md

```diff
@@ -0,0 +1,20 @@
+# Django admin user roles
+
+Roles other than superuser should be defined in authentication and authorization groups in django admin 
+
+## Superuser
+
+Full access
+
+## CISA analyst
+
+### Basic permission level
+
+Staff
+
+### Additional group permissions
+
+auditlog | log entry | can view log entry
+registrar | contact | can view contact
```

Comment:
Since they are going to need to see users too, do you want to add that here or will that be the next PR?

Reply:
I think it's better left for the next PR, alongside the code that implements it?